privaterelay/templatetags/relay_tags.py

```diff
@@ -1,8 +1,9 @@
 from django import template
+from django.conf import settings
 
 register = template.Library()
 
 
 @register.simple_tag
 def remaining_free_aliases(aliases):
-  return 5-len(aliases)
+  return settings.MAX_NUM_BETA_ALIASES-len(aliases)
```

Comment:
Can you make this same change in [`emails.views._index_POST`](https://github.com/mozilla/fx-private-relay/blob/master/emails/views.py#L57) ?